docs/docs/05-reusable-components.md

```diff
@@ -296,6 +296,21 @@ ReactDOM.render(<HelloMessage name="Sebastian" />, mountNode);
 This simplified component API is intended for components that are pure functions of their props. These components must not retain internal state, do not have backing instances, and do not have the component lifecycle methods. They are pure functional transforms of their input, with zero boilerplate.
 However, you may still specify `.propTypes` and `.defaultProps` by setting them as properties on the function, just as you would set them on an ES6 class.
```

Comment:
Let's add a newline above this so it starts a new paragraph and then change the period at the end of the sentence to a colon so it's clearer that the code we're writing is connected to some introduction text.